### ISSUE TITLE
accessibility issue with graphics and sound page

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#1062]
Reported by OU on forums https://www.xerte.org.uk/index.php/nl/forum/accessibility/2661-quizzes-and-audio-player-controls?start=6#7697

With a graphics and sound page set to have the page text aligned left and the image and sound on the right or indeed any of the text align options other than top a screen reader reads the image and then the audio controller before the page text.

I think this is because in the model we have the textHolder div after the imgHolder div so the dom order means the image and sound are read first. However simply moving the textHolder div above the imgHolder div doesn't work and probably due to the current css or script elsewhere in the model.
